client/blocks/comments/post-comment-list.jsx

```diff
@@ -276,7 +277,14 @@ class PostCommentList extends React.Component {
 		const commentText = this.state.commentText;
 
 		// Are we displaying the comment form at the top-level?
-		if ( this.props.activeReplyCommentId && ! this.state.errors ) {
+		if (
+			this.props.activeReplyCommentId ||
+			size(
```

Comment:
why are you doing a scan here instead of (psuedocode):
```js
const comment = this.props.commentsTree[ this.props.activeReplyCommentId ];
if (
  activeReplyCommentId ||
 ( comment && comment.data && comment.data.isPlaceholder && ! comment.data.parent )
)
```